docs/src/usage/analysis.md

```diff
@@ -200,6 +200,41 @@ save("tsa.png", tsa_fig)
 ![Plots of Temporal Sensitivities](/ADRIA.jl/dev/assets/imgs/tsa.png?raw=true "Temporal Sensitivity Analysis")
 
 
+### Convergence Analysis for Sensitivity
+
+When undertaking sensitivity analysis it is importnant to have a sufficient sample size that the sensitivity measure has converged and is relatively stable. To understand convergence behaviour a 
+large sample can be drawn and split into smaller sub-samples. Plotting a sensitivity measure against increasing sample size can help to understand whether the sample size is sufficient for convergence.
+The function `convergence` can be used to calculated a sensitivity measure for an increasing number of samples. The result can then be plotted as band plots or a heat map using `viz.convergence`.
+
+```julia
+
+outcome = ADRIA.mean(ADRIA.metrics.scenario_total_cover(rs); dims=:timesteps)[timesteps=1]
+
+# Plot heat map of sensitivities for increasing number of scenarios for 
+# individual factors (foi).
+foi = [:dhw_scenario, :wave_scenario, :guided]
+Si_N = convergence(scens, outcome, foi)
+ADRIA.viz.convergence(Si_N, Symbol.(foi))
```

Comment:
No indication of where `convergence` comes from?

Is it not within a submodule?
Did you make it so it is not within a submodule and so it leaks into the global namespace?

Reply:
It's inside the ADRIA.sensitivity submodule. I'll add the .sensitivity at the front. I think in the example I copied from I imported the whole model so I didn't need to.